modules/rosa-troubleshooting-deployment.adoc

```diff
@@ -0,0 +1,41 @@
+[id="rosa-troubleshooting-deployment_{context}"]
+= Deployment troubleshooting
+
+
+[id="rosa-troubleshooting-general-deployment-failure_{context}"]
+== General deployment failure
+
+If a cluster deployment fails, the cluster is put into an "error" state.
+
+Run the following command to get more information:
+
+[source,terminal]
+----
+$ rosa describe cluster -c <my_cluster_name> --debug
+----
+
+[id="rosa-troubleshooting-deployment-failure-osdccsadmin_{context}"]
+== Deployment failure with an `osdCcsAdmin` error
+
+If a cluster creation action fails, you can receive the following error message.
+
+.Example output
+[source,terminal]
+----
+Failed to create cluster: Unable to create cluster spec: Failed to get access keys for user 'osdCcsAdmin': NoSuchEntity: The user with name osdCcsAdmin cannot be found.
+----
+
+.Procedure
```

Comment:
I wouldn't use the `.Procedure` section because this isn't a proper procedure module. I'd either use a general introduction statement for this subsection or split the module into two. (Probably the latter.)